src/analytics/private/sqls/redshift/user-m-view-v2.sql

```diff
@@ -0,0 +1,28 @@
+CREATE MATERIALIZED VIEW {{schema}}.user_m_view_v2 
+BACKUP YES 
+DISTSTYLE ALL  
+SORTKEY(user_pseudo_id) 
+AUTO REFRESH YES 
```

Comment:
Let's turn off `AUTO REFERSH`. Trigger the MV refresh in the loading workflow.

Reply:
changed